packages/react/src/components/FluidTextInput/index.js

```diff
@@ -7,3 +7,5 @@
 
 export default from './FluidTextInput';
 export FluidTextInput from './FluidTextInput';
+
+export { default as FluidTextInputSkeleton } from './FluidTextInput.Skeleton';
```

Comment:
I think this needs to be exposed in the main index.js as well and the public api snapshot regenerated.

Reply:
Yea good catch I updated the animations as well @tay1orjones